typesafe_conductr_cli/conduct_info.py

```diff
@@ -18,12 +18,13 @@ def info(args):
     data = [
         {
             'id': bundle['bundleId'],
+            'name': bundle['attributes']['bundleName'],
             'replications': len(bundle['bundleInstallations']),
             'starting': sum([not execution['isStarted'] for execution in bundle['bundleExecutions']]),
             'executions': sum([execution['isStarted'] for execution in bundle['bundleExecutions']])
         } for bundle in json.loads(response.text)
     ]
-    data.insert(0, {'id': 'ID', 'replications': '#REP', 'starting': "#STR", 'executions': '#RUN'})
+    data.insert(0, {'id': 'ID', 'name': 'NAME', 'replications': '#REP', 'starting': "#STR", 'executions': '#RUN'})
```

Comment:
What's the purpose of a leading # like in #REP and #STR? Do we need it for NAME (or ID), too?